workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectReference.cs

```diff
@@ -2,8 +2,35 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     Invokes dispatcher callbacks on GameObjects that represent entities.
+    /// </summary>
     public class GameObjectReference : Component
     {
         public GameObject GameObject;
+
+        public void OnAddComponent(uint componentId)
+        {
+        }
+
+        public void OnRemoveComponent(uint componentId)
+        {
+        }
+
+        public void OnAuthorityChange(uint componentId)
```

Comment:
Can we pass the Authority itself as well here? That shouldn't have boxing issues, and the GameObjectReference itself will need it for injection logic. Alternatively the GameObjectReference could go through the Entity like the ReaderWriter itself will have to, but just passing would be more convenient and readable I think.

Reply:
Yes, we can do that. That would make sense.